src/warp10-datasource.ts

```diff
@@ -296,11 +296,28 @@ export default class Warp10Datasource {
     if (opts.scopedVars) {
       for (let k in opts.scopedVars) {
         let v = opts.scopedVars[k]
-        if (v.selected) {
+        if (v.selected || this.scopedVarIsAll(k)) {
           str += `'${v.value}' '${k}' STORE `
         }
       }
     }
     return str
   }
+
+  /**
+   * Test is a named scoped variable is set to all
+   *
+   * @param name string The name of scoped variable
+   * @return bool If the scoped variable is set to all
+   */
+  private scopedVarIsAll(name): boolean {
```

Comment:
You can add `name` parameter type
`private scopedVarIsAll(name: string): boolean`